src/app/clusters/level-control/level-control.cpp

```diff
@@ -960,18 +960,56 @@ static void moveHandler(app::CommandHandler * commandObj, const app::ConcreteCom
         goto send_default_response;
     }
 
+    // Always validate the rate parameter received. A rate of 0 is invalid.
```

Comment:
I don't think it's the intent of the spec changes that rate == 0 plus mode == stop should stop working...  But maybe it is?  I asked these questions on the spec PR, and we should make sure we don't make changes here until the spec is sorted.

Reply:
The level control movemode enum has no stop. The level control has a separate stop command to stop transitions.

https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/LevelControl.adoc#52-movemodeenum-type

Your comment does applied to color control which I am also working on for a separate pr.

Reply:
Ah, sorry.  Too many things named similarly.  :(